code/modules/cargo/console.dm

```diff
@@ -58,12 +67,91 @@
 	obj_flags |= EMAGGED
 	contraband = TRUE
 
+	space_gear = TRUE
+	heavy_firearms = TRUE
+	medco_trade = TRUE
+	blackmarket = TRUE
+	animal_handing = TRUE
+	adv_sci = FALSE
+
 	// This also permamently sets this on the circuit board
 	var/obj/item/circuitboard/computer/cargo/board = circuit
+
 	board.contraband = TRUE
+	board.space_gear = TRUE
+	board.heavy_firearms = TRUE
+	board.medco_trade = TRUE
+	board.blackmarket = TRUE
+	board.animal_handing = TRUE
+	board.adv_sci = TRUE
+
 	board.obj_flags |= EMAGGED
 	update_static_data(user)
 
+/obj/machinery/computer/cargo/attackby(obj/item/W, mob/living/user, params)
+	var/obj/item/circuitboard/computer/cargo/board = circuit
+	if((istype(W, /obj/item/folder/paperwork_correct/space_gear)) && allowed(user))
```

Comment:
Just make a proc called when hitting the console with a `/obj/item/folder/paperwork_correct` item instead? This is pretty bad.

Reply:
No then it would eat paperwork_correct

Reply:
Make it not eat it then. That's no good excuse for a lengthy else if chain. You even made a cynical screenshot about it.